node/Cargo.toml

```diff
@@ -105,6 +102,14 @@ polkadot-cli = { git = "https://github.com/paritytech/polkadot", branch = "rococ
 polkadot-test-service = { git = "https://github.com/paritytech/polkadot", branch = "rococo-v1" }
 polkadot-parachain = { git = "https://github.com/paritytech/polkadot", branch = "rococo-v1" }
 
+# benchmarking dependencies
+frame-benchmarking = { git = "https://github.com/paritytech/substrate", branch = "rococo-v1" }
+frame-benchmarking-cli = { git = "https://github.com/paritytech/substrate", branch = "rococo-v1" }
```

Comment:
I guess these should be `optional = true` and only depended on when the runtime-benchmarking` feature is enabled.

Reply:
Hmm, guess I'm wrong about at least the frame-benchmarking-cli dependency based on how the substrate node does it.
https://github.com/paritytech/substrate/blob/master/bin/node/cli/Cargo.toml#L100

I was thinking we could feature gate that entire subcommand, but rustc didn't like that. Maybe the `#[cfg(feature = "runtime-benchamrks")]` interfered with the structopt macros? Probably best to just follow Substrate.

Might still be able to make `frame-benchmarking` optional though.

Reply:
Looks like we can make `frame-benchmarking` optional, and I've done that in https://github.com/purestake/moonbeam/commits/joshy-benchmarking-jam I don't fell strongly that we need to especially since Substrate doesn't. I'll resolve this, and let you pull in that commit if you want it.